docs/user-guide/quickstart.md

```diff
@@ -0,0 +1,76 @@
+Quickstart
+==========
+
+Client configuration
+--------------------
+
+Start by importing the Python Client and authentication provider:
+```
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+```
+Next, create an authentication provider and use that to create an authenticated client:
+```
+import os
+
+username = os.environ['TAMR_USERNAME']
+password = os.environ['TAMR_PASSWORD']
+
+auth = UsernamePasswordAuth(username, password)
+tamr = Client(auth)
+```
+Warning:
+
+For security, it's best to read your credentials in from environment variables or secure files instead of hardcoding them directly into your code.
+
+For more, see [User Guide > Secure Credentials](secure-credentials.html).
```

Comment:
Links to other docs should be done without extensions:

```suggestion
For more, see [User Guide > Secure Credentials](secure-credentials).
```

Right now CI is flagging this as the cause of a [failed check](https://github.com/Datatamer/tamr-client/pull/332/checks?check_run_id=426331717) when trying to build the docs.

FYI: You can run `poetry run invoke docs` locally to make sure the docs are built correctly and look correct! (https://tamr-client.readthedocs.io/en/latest/contributor-guide/run-and-build.html#docs)